contracts/bns/BNSRegistrar.sol

```diff
@@ -12,7 +13,8 @@ contract BNSRegistrar is ERC721, IBaseRegistrar, Ownable {
     BNS public bns;
     // The Reverse Registrar
     ReverseRegistrar public immutable reverseRegistrar;
-    // The namehash of the TLD this registrar owns (eg, .eth)
+
+    // The namehash of the TLD this registrar owns (eg, .b)
```

Comment:
Confused here? What is the namehash of the TLD? `baseNode`? Where is that set?

Reply:
Set at construction time, updated comment to clarify